Controller/Payment/Webhook.php

```diff
@@ -555,4 +558,19 @@ protected function setWebhookNotifiedAt($entity_id)
         $order->setRzpWebhookNotifiedAt(time());
         $order->save();
     }
+
+    protected function setWebhookData($entityId, $webhookVerifiedStatus, $paymentId, $event, $amount)
+    {
+        $webhookData = array(
+            "webhook_verified_status"   => $webhookVerifiedStatus,
+            "payment_id"                => $paymentId,
+            "event"                     => $event,
+            "amount"                    => $amount 
+        );
+        $webhookDataText = serialize($webhookData);
```

Comment:
@nikhilsharma1100 Please add this on new line.

Reply:
okay